src/PrestaShopBundle/Model/Product/AdminModelAdapter.php

```diff
@@ -193,7 +196,8 @@ public function __construct(
         FeatureDataProvider $featureDataProvider,
         PackDataProvider $packDataProvider,
         ShopContext $shopContext,
-        TaxRuleDataProvider $taxRuleDataProvider
+        TaxRuleDataProvider $taxRuleDataProvider,
+        Router $router
```

Comment:
Recurrent question these days.. does this count as a BC break?

Reply:
Broken compatibility added in the PR description